apis/placement/v1beta1/binding_types.go

```diff
@@ -263,6 +282,30 @@ func (b *ClusterResourceBinding) GetCondition(conditionType string) *metav1.Cond
 	return meta.FindStatusCondition(b.Status.Conditions, conditionType)
 }
 
+// GetBindingSpec returns the binding spec.
+func (b *ClusterResourceBinding) GetBindingSpec() *ResourceBindingSpec {
+	return &b.Spec
+}
+
+// SetBindingSpec sets the binding spec.
+func (b *ClusterResourceBinding) SetBindingSpec(spec *ResourceBindingSpec) {
```

Comment:
should we use the deep copy in case there are the multiple callers updating the same spec?